docs/compiler.md

```diff
@@ -38,7 +38,7 @@ a dialect of Go rather than a complete port of the language:
  * arrays (`[4]byte`) are not supported (https://github.com/nspcc-dev/neo-go/issues/3524)
  * `min()` and `max()` are supported for integer types only.
  * `clear()` is not supported (https://github.com/nspcc-dev/neo-go/issues/3091)
- * ranging over integers in `for` is not supported (https://github.com/nspcc-dev/neo-go/issues/3525)
+ * ranging over integers in `for` is supported.
```

Comment:
Lines like this can just be removed. We're not describing Go features here, we're telling about differences in our implementation of Go. No difference --- nothing to talk about.